ops/trim.py

```diff
@@ -5,6 +5,7 @@
 for file in files:
     newAudio = AudioSegment.from_wav(file)[0:10000]  # milliseconds
     # exports to a .wav file to a different path
-    newAudio.export(file.rsplit('\\', 1)[0]+"/new/"+file.rsplit('\\', 1)[1], format="wav")
+newAudio.export(file.rsplit('\\', 1)[0]+"/new/"+file.rsplit('\\', 1)[1], format="wav")
```

Comment:
indenting is off here, but i think i messed that up. please indent the last line one time